posthog/tasks/status_report.py

```diff
@@ -74,19 +89,21 @@ def status_report(*, dry_run: bool = False) -> Dict[str, Any]:
             # Dashboards
             team_dashboards = Dashboard.objects.filter(team=team).exclude(deleted=True)
             team_report["dashboards_count"] = team_dashboards.count()
+            org_usage_summary["dashboards_created"] += team_report["dashboards_count"]
             team_report["dashboards_template_count"] = team_dashboards.filter(creation_mode="template").count()
             team_report["dashboards_shared_count"] = team_dashboards.filter(is_shared=True).count()
             team_report["dashboards_tagged_count"] = team_dashboards.exclude(tags=[]).count()
 
             # Feature Flags
             feature_flags = FeatureFlag.objects.filter(team=team).exclude(deleted=True)
             team_report["ff_count"] = feature_flags.count()
+            org_usage_summary["feature_flags_created"] += team_report["ff_count"]
```

Comment:
Perhaps worth keeping the name same even if it's bad? :) Just for sake of easy analysis